model/src/test/kotlin/OrtResultTest.kt

```diff
@@ -270,7 +270,7 @@ class OrtResultTest : WordSpec({
     }
 
     "getRuleViolations()" should {
-        "return unfiltered rule violations if omitResolved is false" {
+        "return the issues, even tough it is resolved and a hint" {
```

Comment:
These are rule violations, not issues. To me, the new name is less readable as it does not refer to `omitResolved`, so it sounds as if resolved violations should be returned unconditionally.

Reply:
Do you have a proposal ?